test/utils.jl

```diff
@@ -149,4 +149,68 @@ end
     @test_throws AssertionError DataFrames.split_to_chunks(10, 11)
 end
 
+@testset "_findall(B::BitVector)" begin
+    Random.seed!(1234)
```

Comment:
we need to add `using Random` on top of the module - I thought was already added.

Reply:
Sure